Makefile

```diff
@@ -396,3 +397,130 @@ image-mnist-job-test-build: ## Build container image with the MNIST job.
+	-make delete-ai-platform-operator
+	-make delete-nvidia-operator
+
+##@ general
+.PHONY: delete-ai-platform-operator
+delete-ai-platform-operator:
+ifeq ($(USE_RHOAI), true) ## Delete RHOAI Operator
+	-make delete-rhoai-operator
+else ## Delete Open Data Hub Operator
+	-make delete-opendatahub-operator
+endif
+
+.PHONY: install-ai-platform-operator
+install-ai-platform-operator:
+ifeq ($(USE_RHOAI), true) ## Delete RHOAI Operator
+	-make install-rhoai-operator
+else ## Delete Open Data Hub Operator
+	-make install-opendatahub-operator
+endif
+
+.PHONY: delete-rhoai-operator
+delete-rhoai-operator: ## Delete RHOAI Operator
+	@echo -e "\n ==> Deleting OpenShift AI Operator \n"
+	-kubectl delete subscription rhods-operator -n redhat-ods-operator
+	-export CLUSTER_SERVICE_VERSION=`kubectl get clusterserviceversion -n redhat-ods-operator -l operators.coreos.com/rhods-operator.redhat-ods-operator -o custom-columns=:metadata.name`; \
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n redhat-ods-operator
+	kubectl delete namespace redhat-ods-operator
+
+.PHONY: install-rhoai-operator
+install-rhoai-operator: ## Install RHOAI Operator
+	@echo -e "\n ==> Installing OpenShift AI Operator \n"
+	-kubectl create ns redhat-ods-operator
+	kubectl create -f contrib/configuration/rhoai/rhoai-operator-subscription.yaml
+	@echo Waiting for rhoai-operator Subscription to be ready
+	kubectl wait -n redhat-ods-operator subscription/rhods-operator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
+	@echo -e "\n==> Creating default Data Science Cluster \n"
+	kubectl apply -f contrib/configuration/rhoai/default-dsci.yaml --server-side
+	kubectl apply -f contrib/configuration/rhoai/default-dsc.yaml --server-side
+
+.PHONY: delete-opendatahub-operator
+delete-opendatahub-operator: ## Delete OpenDataHub operator
+	@echo -e "\n==> Deleting OpenDataHub Operator \n"
+	-kubectl delete subscription opendatahub-operator -n openshift-operators
+	-export CLUSTER_SERVICE_VERSION=`kubectl get clusterserviceversion -n openshift-operators -l operators.coreos.com/opendatahub-operator.openshift-operators -o custom-columns=:metadata.name`; \
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n openshift-operators
+	-kubectl delete namespace opendatahub
+
+.PHONY: install-opendatahub-operator
+install-opendatahub-operator: ## Install OpenDataHub operator
+	@echo -e "\n==> Installing OpenDataHub Operator \n"
+	-kubectl create ns opendatahub
+	kubectl create -f contrib/configuration/odh/opendatahub-operator-subscription.yaml
+	@echo Waiting for opendatahub-operator Subscription to be ready
+	kubectl wait -n openshift-operators subscription/opendatahub-operator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
+	sleep 2
+	kubectl wait --for=condition=available deployment/opendatahub-operator-controller-manager -n openshift-operators --timeout=180s
+	kubectl apply -f contrib/configuration/odh/default-dsci.yaml --server-side
```

Comment:
Got a failure here:
```
[ksuta@ksuta-thinkpadp1gen4i codeflare-operator]$ make install-ai-platform-operator 
make install-opendatahub-operator
make[1]: Entering directory '/home/ksuta/src/github.com/project-codeflare/codeflare-operator'

==> Installing OpenDataHub Operator 

kubectl create ns opendatahub
namespace/opendatahub created
kubectl create -f contrib/configuration/odh/opendatahub-operator-subscription.yaml
subscription.operators.coreos.com/opendatahub-operator created
Waiting for opendatahub-operator Subscription to be ready
kubectl wait -n openshift-operators subscription/opendatahub-operator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
subscription.operators.coreos.com/opendatahub-operator condition met
sleep 2
kubectl wait --for=condition=available deployment/opendatahub-operator-controller-manager -n openshift-operators --timeout=180s
deployment.apps/opendatahub-operator-controller-manager condition met
kubectl apply -f contrib/configuration/odh/default-dsci.yaml --server-side
Error from server (InternalError): Internal error occurred: failed calling webhook "operator.opendatahub.io": failed to call webhook: Post "https://opendatahub-operator-controller-manager-service.openshift-operators.svc:443/validate-opendatahub-io-v1?timeout=10s": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: current time 2024-05-15T06:37:37Z is before 2024-05-15T06:37:52Z
```

Seems ODH operator needs time to refresh its certificates.